cypress/integration/page.spec.js

```diff
@@ -975,7 +1053,7 @@
 
     it('should show only email error if email is empty', () => {
       newCommentForm.nameInput().type('Some name');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
Consider using a more descriptive comment body for the test input to improve readability.